src/components/Dashboard.js

```diff
@@ -29,8 +29,8 @@ function getStepName(stepKey, state) {
 const guideSteps = [
   { key: "general", StepContent: GeneralSettings },
   { key: "products", StepContent: ManageProducts },
-  { key: "customize", StepContent: CustomizeStore },
-  { key: "advanced", StepContent: AdvancedFeatures },
+  { key: "pages", StepContent: CustomizeStore },
+  { key: "additional", StepContent: AdvancedFeatures },
```

Comment:
I think we can name it `features` instead

Reply:
done